astropy/modeling/polynomial.py

```diff
@@ -1,6 +1,50 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
 This module contains models representing polynomials and polynomial series.
+
+Notes regarding usage of domain and window
```

Comment:
This text is not exposed anywhere for users. It's a bit verbose for docstrings, but maybe it can be included in the narrative docs, somewhere?

Reply:
I'll see if I can find a better place.